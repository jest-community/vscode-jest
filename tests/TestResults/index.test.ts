jest.disableAutomock()

describe('TestResults', () => {
  describe('module.exports', () => {
    it('should match the snapshot', () => {
      expect(require('../../src/TestResults')).toMatchSnapshot()
    })
  })
})
