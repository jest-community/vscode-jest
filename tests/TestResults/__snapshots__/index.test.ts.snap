// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TestResults module.exports should match the snapshot 1`] = `
Object {
  "TestReconciliationState": Object {
    "KnownFail": "KnownFail",
    "KnownSkip": "KnownSkip",
    "KnownSuccess": "KnownSuccess",
    "Unknown": "Unknown",
  },
  "TestResultProvider": [Function],
  "resultsWithLowerCaseWindowsDriveLetters": [Function],
}
`;
